script.js

```diff
@@ -0,0 +1,254 @@
+
+const projectSeeLiveSource = document.createElement('div');
+projectSeeLiveSource.className = 'projectSee';
+projectSeeLiveSource.innerHTML = `
+<button class="see-live close-project" type="button" onclick="window.location.href='https://d2ndjim.github.io/PortFolio/'">
+<p>See live</p><img src="images/ic_link.svg" alt="download-icon">
+</button>
+<button class="see-source close-project" type="button" onclick="window.location.href='https://github.com/d2ndjim/PortFolio'">
+    <p>See source</p><img src="images/ic_github_white.svg" alt="github">
+</button>
+`;
+
+const style = document.createElement('style');
+style.innerHTML = `
+    .show-popup {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        background-color: white;
+        color: #1A2236;
+        width: 100vw;
+        height: 100vh;
+        position: fixed;
+        z-index: 10;
+        top: -100%;
+        overflow-y: scroll !important;
+        max-height: 100vh;
+        font-family: 'Poppins', sans-serif;
+        font-size: 6vw;
+        padding-inline-start: 6vw;
+    }
+
+    .show-popup .popUpTop {
+        display: flex;
+        flex-direction: row-reverse;
+        width: 90vw;
+        justify-content: space-between;
+    }
+
+    .show-popup h2 {
+        margin-bottom: 0.5rem;
+    }
+    
+    .close-project {
+        align-self: flex-start;
+        margin: 1.5rem 1.5rem 0.5rem 0;
+    }
+    
+    .show-popup .technologies {
+        display: flex;
+        align-items: center;
+        padding-inline-start: 0;
+        justify-content: space-between;
+        height: 10vw;
+        width: 85vw;
+        font-weight: 300;
+    }
+
+    .show-popup .technologies li {
+        list-style: none;
+        text-align: center;
+        padding-top: 0.7em;
+        padding-bottom: 0.7em;
+        white-space: nowrap;
+        border: solid black 1px;
+        font-size: 3vw;
+        color: #1a2236;
+        width: 25vw;
+    }
+
+    .show-popup .thumbnail {
+        width: 88vw;
+        min-height: 53vw;
+        box-shadow: 0 0 50px 5px black inset;
+    }
+
+    .show-popup .two {
+        background-image: url(../images/project_thumbnail_2.svg);
+    }
+
+    .show-popup .three {
+        background-image: url(../images/project_thumbnail_3.svg);
+    }
+
+    .show-popup .four {
+        background-image: url(../images/project_thumbnail_4.svg);
+    }
+
+    .show-popup .five {
+        background-image: url(../images/project_thumbnail_5.svg);
+    }
+
+    .show-popup .six {
+        background-image: url(../images/project_thumbnail_6.svg);
+    }
```

Comment:
- [ ] Kindly there's a problem with the pictures, it only appears with project one but the rest no. 

![image](https://user-images.githubusercontent.com/35535130/156370397-2bcf8618-9fa7-4210-a849-1a00863201d7.png)